### PR TITLE
Add UriAsserts

### PR DESCRIPTION
**Pull request by MichaelEvans**

### CODE REVIEW

File: assertj-android/src/main/java/org/assertj/android/api/net/UriAssert.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package org.assertj.android.api.net;
+
+import android.net.Uri;
+
+import org.assertj.core.api.AbstractAssert;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UriAssert extends AbstractAssert<UriAssert, Uri> {
```
Comment (JakeWharton) [inline assertj-android/src/main/java/org/assertj/android/api/net/UriAssert.java]: final

**Comment by JakeWharton** [comment]
Comment when updated. Don't forget to squash.

**Comment by MichaelEvans** [comment]
@JakeWharton done!

**Comment by JakeWharton** [comment]
Put a release on my TODO list. Probably Tuesday unless I'm bored over the long weekend.

**Pull request merged**